Classes/Tree/NodeRepository.php

```diff
@@ -172,7 +172,7 @@ protected function retrieveByNamespace($namespace, $respectEnableFields)
         /*
          * RespectEnableFields = FALSE means, that all records are selected INCLUDING the deleted records
          */
-        if (!$respectEnableFields) {
+        if (!$respectEnableFields    && !$this->treeContext->isIncludeDeleted()) {
```

Comment:
Hey Kristin, ist die Negation nicht falsch. Und sollte das nicht sogar: 
` if (!$respectEnableFields || $this->treeContext->isIncludeDeleted())` heißen?